apps/server/package.json

```diff
@@ -13,7 +13,17 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "ts-node-dev": "^2.0.0"
+    "@typescript-eslint/eslint-plugin": "^6.7.0",
```

Comment:
And please add npm script to lint in your scripts, then we can have a chance to check if linting rules really does work.